scripts/netbios-interfaces.nse

```diff
@@ -0,0 +1,57 @@
+local shortport = require "shortport"
+local netbios = require "netbios"
+local string = require "string"
+local stdnse = require "stdnse"
+
+description = [[
+Attempts to retrieve via NetBIOS the target's network interfaces.
+Additional network interfaces may reveal more information about target.
+In particular, it is very useful for finding paths to non-routed networks if target has more than one NIC.
+]]
+
+---
+-- @usage
+-- nmap -sU --script netbios-interfaces.nse -p 137 <host>
+--
+-- @output
+-- PORT    STATE SERVICE
+-- 137/udp open  netbios-ns
+-- | netbios-interfaces: 
+-- |   hostname: NOTEBOOK-NB3
+-- |   interfaces: 
+-- |     192.168.128.100
+-- |     172.24.80.1
+-- |     172.27.96.1
+-- MAC Address: 9C:7B:EF:AA:BB:CC (Hewlett Packard)
+
```

Comment:
Please add also section `@xmloutput`, which is just the XML output of the same sample scan.

Reply:
\+

Reply:
You have forgotten to add `<elem key="hostname">NOTEBOOK-NB3</elem>` but I will take care of it.